corehq/apps/es/ledgers.py

```diff
@@ -0,0 +1,48 @@
+"""
+LedgerES
+------
+.. code-block:: python
+
+    from corehq.apps.es import cases as case_es
+
+    q = (ledgers.LedgerES()
+         .domain('testproject')
+         .section('stock')
+"""
+from .es_query import HQESQuery
+from . import filters
+
+
+class LedgerES(HQESQuery):
+    index = 'ledgers'
+
+    @property
+    def builtin_filters(self):
+        return [
+            case,
+            section,
+            entry,
+            location,
+            modified_range,
+            filters.domain,
+        ]
```

Comment:
@snopoke Any reason ```builtin_filters``` doesn't include ```super(LedgerES, self).builtin_filters```?  I want to use ```filters.NOT```, which isn't included here.

Reply:
@nickpell nope, just an oversight on my part